subprojects/testing-jvm/src/integTest/groovy/org/gradle/testing/junit/JUnitLoggingOutputCaptureIntegrationTest.groovy

```diff
@@ -93,19 +91,19 @@ public class OkTest {
 Test class OkTest -> before class out
 Test class OkTest -> before class err
 Test class OkTest -> test constructed
-Test anotherOk(OkTest) -> before out
-Test anotherOk(OkTest) -> before err
-Test anotherOk(OkTest) -> ok out
-Test anotherOk(OkTest) -> ok err
-Test anotherOk(OkTest) -> after out
-Test anotherOk(OkTest) -> after err
+Test ${testName('anotherOk')}(OkTest) -> before out
```

Comment:
❌ Could we overwrite "outputContains" in the base class instead of changing every assertion? Paremeterized assertions can be really hard to read.

Reply:
Done.